### DISCUSSION TITLE
Mention notif appears each time I enter to a playlist lobby

### DISCUSSION THREAD

**Discussion by Ladwick** [ppy/osu#23382] [Q&A] [answered]
### Type

Cosmetic

### Bug description

No matter how many times I dismiss it, the notification will keep appearing everytime I enter the lobby, however, if I play a map and then go back to the lobby, the notif won't appear, this just happens if I leave the playlist and enter again.

### Screenshots or videos

https://user-images.githubusercontent.com/37979533/235835369-fb6bfdb2-e60a-4689-8819-698f60e060b0.mp4

### Version

2023.419.0

### Logs

not needed

**Comment by peppy** [comment]
This seems like correct behaviour. Not sure how we can improve this too much.

**Reply by Theighlin** [reply]
Should probably notify again only if mentioed in new messages since user last joined?

**Reply by Ladwick** [reply]
Oh sorry, I didn't know it was inteded but (in my opinion) I think it's a little confusing to be pinged about the same message multiple times, it is possible to ping the user once per mention? A behaviour that feels more natural to me is it notifying you about a mention and dismissing it for good once you have seen the message and will notify you if you are mentioned ever again instead of doing it over the same message, I'm not sure if this is possible tho, but the idea is there.

**Reply by peppy** [reply, marked as answer]
Probably the sanest way would be to keep a list of already-notified message ids permanently at a high level. I wouldn't be against this.

**Reply by Joehuu** [reply]
Recently opened as an issue: https://github.com/ppy/osu/issues/24715
